fixtures/flight/src/AddTodo.client.js

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react';
+
+import Container from './Container.js';
+import useMutation from './useMutation.client';
+
+export default function AddTodo() {
+  const [text, setText] = React.useState('');
+  const [postTodo, isPosting] = useMutation(async () => {
+    setText('');
+    await fetch('http://localhost:3001/todos', {
+      method: 'POST',
+      mode: 'cors',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({text}),
+    });
```

Comment:
You can have this fetch return a new Flight response. It means you'll have to hard code *which* Flight root entry point this corresponds to. That's kind of a hard problem for a router but it's good to indicate that it is a problem that's worth solving.